README.md

```diff
@@ -190,6 +190,7 @@ $ gcloc languages
   Plain Text       |  .txt .text    |                 |
   PowerShell       |  .ps1          |  #              |
   Processing       |  .pde          |  //             |  /* */
+  PRQL             |  .prql         |  #              |
   Protocol Buffers |  .proto        |  //             |
   Python           |  .py           |  #              |  """ """
```

Comment:
You added between `Processing` and `Protocol Buffers`, could you add `PRQL` between `Protocol Buffers` and  `Python` ?
```suggestion
  Processing       |  .pde          |  //             |  /* */
  Protocol Buffers |  .proto        |  //             |
  PRQL             |  .prql         |  #              |
  Python           |  .py           |  #              |  """ """
```